subprojects/plugin-use/src/integTest/groovy/org/gradle/plugin/use/VersionedPluginUseIntegrationTest.groovy

```diff
@@ -33,6 +33,8 @@ class VersionedPluginUseIntegrationTest extends AbstractIntegrationSpec {
     MavenHttpPluginRepository pluginRepo = MavenHttpPluginRepository.asGradlePluginPortal(executer, mavenRepo)
 
     def setup() {
+        // https://github.com/gradle/build-tool-flaky-tests/issues/49
```

Comment:
could you include some of the detail from Jendrik's comment here? https://github.com/gradle/build-tool-flaky-tests/issues/49#issuecomment-581326278

Reply:
Added. Thanks.